### ISSUE TITLE
Could not load nightfox lightline colorscheme in Vim

### ISSUE THREAD

**Issue by JS-Zheng** [EdenEast/nightfox.nvim#206]
## Description
After nightfox has been [rewritten](https://github.com/EdenEast/nightfox.nvim/pull/77), support for [lightline.vim](https://github.com/itchyny/lightline.vim) in Vim is broken; however, the support in NeoVim works well.

```vim
:messages
lightline.vim: Could not load colorscheme nightfox.
```

## Additional info / context
```
VIM - Vi IMproved 9.0 (2022 Jun 28, compiled Aug 06 2022 10:35:28)
macOS version - x86_64
Included patches: 1-150
Compiled by Homebrew

+lua 
+statusline
```

**Comment by nullchilly** [comment]
@EdenEast Because in vim `luaeval` only works for table with depth = 1

Proposed solution: https://github.com/catppuccin/nvim/blob/main/autoload/lightline/colorscheme/catppuccin.vim

**Comment by nullchilly** [comment]
@EdenEast Can confirm it works now 👍🏻

**Comment by EdenEast** [comment]
Sorry this issue is resolved by #269.
